src/components/CopyableAccountNumber/index.tsx

```diff
@@ -15,7 +15,7 @@ interface ComponentProps {
 
 const CopyableAccountNumber: FC<ComponentProps> = ({accountNumber, className, isCopyButtonAtBottom}) => {
   const handleCopy = (): void => {
-    displayToast('Account Number copied to the clipboard', 'success');
+    displayToast('Account Number copied to the clipboard', 'success', 'copyable');
```

Comment:
Why did you pass a `className`? 🤔

Reply:
Ah, my bad I was using that to test the behavior of the `displayToast` function and forgot to remove it 😅